Proton/Sources/EditorCommand/Commands/ListCommand.swift

```diff
@@ -88,11 +88,14 @@ public class ListCommand: EditorCommand {
         }
 
         guard let attrValue = attributeValue else {
-            let paragraphStyle = editor.paragraphStyle
+            let paragraphStyle = editor.paragraphStyle.mutableParagraphStyle
+            paragraphStyle.firstLineHeadIndent = 0
             editor.addAttributes([
                 .paragraphStyle: paragraphStyle
             ], at: selectedRange)
+            editor.typingAttributes[.paragraphStyle] = paragraphStyle
             editor.removeAttribute(.listItem, at: selectedRange)
+            editor.typingAttributes[.listItem] = nil
```

Comment:
This resolves the issue where removing a list item between two regular paragraphs would not remove the indentation.